lib/ansible/plugins/strategy/__init__.py

```diff
@@ -367,14 +367,17 @@ def get_original_host(host_name):
             else:
                 return self._inventory.get_host(host_name)
 
+        def get_handler_templar(handler_task):
```

Comment:
Does this need to be a local function?  Several of the others are, because they are using context from `_process_pending_results`.  I don't think this one needs to though, it would just require passing in `iterator` also.

Not a blocker, just a potential improvement.

Reply:
Thanks for the suggestion, fixed that.